<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeftTool</class>
 <widget class="QWidget" name="LeftTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>408</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#LeftTool {
	background:transparent;
}
QToolButton {
	background:transparent; 
	border: 0px;
	color:#888888;
	font: 18px &quot;PingFang SC-Medium&quot;;
}
QLabel {
	background:transparent;
	color:white;
	font: 14px &quot;PingFang SC-Medium&quot;;
}
QFrame#frame_brush_size,#frame_brush_alpha {
	background: rgba(36,36,36,0.9);
	border-radius: 8px 8px 8px 8px;
}
QFrame#left_tool_set {
	background: #121212;
}

QSlider::handle:vertical { 
	background:#A2A2A2;
	border-radius: 2px;
	height: 12px; 
	margin: 2px; 
} 
QSlider::handle:vertical:hover {
	background: #FFFFFF;
}
QSlider::groove:vertical { 
	background:#242424;
	width:28px;
} </string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="left_tool_set">
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QSlider" name="verticalSlider_brush">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>152</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>152</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QtToolButtonEx" name="toolButton_picker">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>〇</string>
        </property>
        <property name="icon">
         <iconset resource="../extres.qrc">
          <normaloff>:/ResExt/mipmap-hdpi/picker_0.png</normaloff>:/ResExt/mipmap-hdpi/picker_0.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="verticalSlider_alpha">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>152</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>152</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QtToolButtonEx" name="toolButton_undo">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>撤销</string>
        </property>
        <property name="icon">
         <iconset resource="../extres.qrc">
          <normaloff>:/ResExt/mipmap-mdpi/undo_0.png</normaloff>:/ResExt/mipmap-mdpi/undo_0.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QtToolButtonEx" name="toolButton_redo">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>重做</string>
        </property>
        <property name="icon">
         <iconset resource="../extres.qrc">
          <normaloff>:/ResExt/mipmap-mdpi/redo_0.png</normaloff>:/ResExt/mipmap-mdpi/redo_0.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>152</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_brush_size">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>0</y>
        <width>152</width>
        <height>152</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>152</width>
        <height>152</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>152</width>
        <height>152</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>18</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>18</number>
       </property>
       <item>
        <widget class="QLabel" name="label_brush_size_text">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>尺寸(10%)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_brush_size_show">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_brush_alpha">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>190</y>
        <width>152</width>
        <height>152</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>152</width>
        <height>152</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>152</width>
        <height>152</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>18</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>18</number>
       </property>
       <item>
        <widget class="QLabel" name="label_brush_alpha_text">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>不透明度(30%)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_brush_alpha_show">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtToolButtonEx</class>
   <extends>QToolButton</extends>
   <header location="global">Utils/qttoolbuttonex.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../extres.qrc"/>
 </resources>
 <connections/>
</ui>
